ISISNeutronMuon/MDANSE
# GUI code cleanup

**Description of work**
This PR mainly removes files which were added in the past but ended up not being necessary as other solutions were being gradually implemented.

**Fixes**
1. Removed InteractiveConverter,
2. Removed Dialog/Wizard code meant to be used with InteractiveConverter,
3. Deleted models from the prototype GUI (before the tabbed GUI was created),
4. Deleted unused widgets (oscillating progress bar),
5. Deleted parts of the Molecular Viewer which we don't use (e.g. residues database, since we have one in the main MDANSE already.)

**To test**
All tests must pass.
Try loading large structures (e.g. a protein) into the Molecular Viewer to check that nothing got broken when the residues database was removed.
